lib/core/src/Cardano/Wallet/Api/Types.hs

```diff
@@ -1033,6 +1046,7 @@ data ApiValidityBound
 data ApiSignTransactionPostData = ApiSignTransactionPostData
     { transaction :: !(ApiT SealedTx)
```

Comment:
Btw, `ApiT` isn't a strict container, so `!(ApiT !SealedTx)` makes more sense.

Reply:
As @HeinrichApfelmus  explained, `ApiT` is declared as a `newtype` and those guys are strict by their nature. So my comment doesn't apply.